File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/nuro/OUT0106U00PatientItemInfo.java
package nta.med.data.model.ihis.nuro;

public class OUT0106U00PatientItemInfo {
	private String patientInfo;
    private String patientInfoName;
    private String contKey;
	public OUT0106U00PatientItemInfo(String patientInfo,
			String patientInfoName, String contKey) {
		super();
		this.patientInfo = patientInfo;
		this.patientInfoName = patientInfoName;
		this.contKey = contKey;
	}
	public String getPatientInfo() {
		return patientInfo;
	}
	public void setPatientInfo(String patientInfo) {
		this.patientInfo = patientInfo;
	}
	public String getPatientInfoName() {
		return patientInfoName;
	}
	public void setPatientInfoName(String patientInfoName) {
		this.patientInfoName = patientInfoName;
	}
	public String getContKey() {
		return contKey;
	}
	public void setContKey(String contKey) {
		this.contKey = contKey;
	}
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Outs/OUT0106U00PatientItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Outs
{
    [Serializable]
	public class OUT0106U00PatientItemInfo
	{
		private String _patientInfo;
		private String _patientInfoName;
		private String _contKey;

		public String PatientInfo
		{
			get { return this._patientInfo; }
			set { this._patientInfo = value; }
		}

		public String PatientInfoName
		{
			get { return this._patientInfoName; }
			set { this._patientInfoName = value; }
		}

		public String ContKey
		{
			get { return this._contKey; }
			set { this._contKey = value; }
		}

		public OUT0106U00PatientItemInfo() { }

		public OUT0106U00PatientItemInfo(String patientInfo, String patientInfoName, String contKey)
		{
			this._patientInfo = patientInfo;
			this._patientInfoName = patientInfoName;
			this._contKey = contKey;
		}

	}
}